docs/rfcs/0007-deployment-chain.md

```diff
@@ -0,0 +1,129 @@
+- Start Date: 2021-11-03
+- Target Version: 0.22.0
+
+# Summary
+This RFC introduces a new way to enable users to deploy their complex system (or applications) in a more flexible way by manipulating the currently supporting application's deployment.
+
+# Motivation
+With the current supporting deployment, our users have to define their application as one of the supporting application's kinds (K8s, Terraform, CloudRun, Lambda, ECS), they can still define multiple PipeCD applications as parts of their application and trigger them one by one, but it's a bit difficult for them to control the deployment chain smoothly.
+
+![image](assets/deployment-chain-kind.png)
+
+Another typical usecase is when users want to build up some kind of deployment chain such as deploy application to the development environment, then if it's done successfully, deploy that application to the staging and then the production environment.
```

Comment:
nit: because many "deploy", "deployment", "development" makes it hard to read.

```suggestion
Another typical use-case is when users want to build up some kind of chain such as deploy application to the development environment, then if it's done successfully, deploy that application to the staging and then the production environment.
```